classes/Product.php

```diff
@@ -131,21 +116,6 @@ class ProductCore extends ObjectModel
     /** @var string Reference */
     public $reference;
 
-    /**
-     * @var string Supplier Reference
-     *
-     * @deprecated since 1.7.7.0
```

Comment:
Same I think we should not remove things deprecated in 1.7.7 it's too early

We need to think about making the transition smoother to 8.0.0 😉 this is to ease adoption